### PR TITLE
L1T DQM Plots for Kalman Filter for BMTF  - backport - CMSSW_10_1_X

### PR DESCRIPTION
**Pull request by ghost**
New DQM Plots for Kalman Filter. The whole task is based on [Kalman DQM BMTF](https://its.cern.ch/jira/browse/CMSLITDPG-609)

### CODE REVIEW

**Comment by dtsitson** [comment]
backport of #23389

**Comment by thomreis** [comment]
please test

File: DQM/L1TMonitor/python/L1TStage2Emulator_cff.py [hunk 1]
```diff
@@ -88,11 +99,15 @@
 
 # CaloLayer2
 from DQM.L1TMonitor.L1TdeStage2CaloLayer2_cfi import *
+from DQM.L1TMonitor.L1TStage2CaloLayer2_cfi import *
```
Comment (thomreis) [inline DQM/L1TMonitor/python/L1TStage2Emulator_cff.py]: Please remove the calo layer2 related code changes.

File: DQM/L1TMonitor/python/L1TStage2Emulator_cff.py [hunk 2]
```diff
@@ -122,7 +138,10 @@
 # sequence to run only for validation events
 l1tStage2EmulatorOnlineDQMValidationEvents = cms.Sequence(
     l1tdeStage2CaloLayer1 +
+    # We process both layer2 and layer2emu in same sourceclient
```
Comment (thomreis) [inline DQM/L1TMonitor/python/L1TStage2Emulator_cff.py]: Please remove the calo layer2 related code changes.

File: DQM/L1TMonitor/python/L1TStage2Emulator_cff.py [hunk 3]
```diff
@@ -122,7 +138,10 @@
...
+    # to be able to divide them in the MonitorClient
     l1tdeStage2CaloLayer2 +
+    l1tStage2CaloLayer2 +
```
Comment (thomreis) [inline DQM/L1TMonitor/python/L1TStage2Emulator_cff.py]: Please remove the calo layer2 related code changes.

File: DQM/L1TMonitorClient/python/L1TStage2EmulatorMonitorClient_cff.py [hunk 4]
```diff
@@ -17,6 +17,7 @@
 from DQM.L1TMonitorClient.L1TStage2EmulatorQualityTests_cff import *
 
 # Calo trigger layer2 client
+from DQM.L1TMonitorClient.L1TStage2CaloLayer2DEClient_cfi import *
```
Comment (thomreis) [inline DQM/L1TMonitorClient/python/L1TStage2EmulatorMonitorClient_cff.py]: Please remove the calo layer2 related code changes.

File: DQM/L1TMonitorClient/python/L1TStage2EmulatorMonitorClient_cff.py [hunk 5]
```diff
@@ -43,9 +47,11 @@
 
 # L1T monitor client sequence (system clients and quality tests)
 l1TStage2EmulatorClients = cms.Sequence(
-		        l1tStage2CaloLayer2DEClientSummary
+                        l1tStage2CaloLayer2DEClient
```
Comment (thomreis) [inline DQM/L1TMonitorClient/python/L1TStage2EmulatorMonitorClient_cff.py]: Please remove the calo layer2 related code changes.

**Review by thomreis — CHANGES_REQUESTED** [review]
There are some calo layer 2 related changes that are not part of the master PR and that should be removed.

**Comment by thomreis** [comment]
+1

**Comment by andrius-k** [comment]
+1

**Pull request closed without merging**